.github/workflows/detect-duplicate.yml

```
@@ -20,14 +20,15 @@ jobs:
       issues: write
       contents: read
     steps:
-      - name: 📥 Checkout repository
-        uses: actions/checkout@9c091bb21b7c1c1d1991bb908d89e4e9dddfe3e0 # v7.0.0
+      - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)